code/game/machinery/vending_types.dm

```diff
@@ -108,15 +108,17 @@
 	products = list(
 		/obj/item/weapon/reagent_containers/food/drinks/coffee = 25,
 		/obj/item/weapon/reagent_containers/food/drinks/tea = 25,
-		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 25
+		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 25,
+		/obj/item/weapon/reagent_containers/food/snacks/donut/normal = 15
 	)
 	contraband = list(
 		/obj/item/weapon/reagent_containers/food/drinks/ice = 10
 	)
 	prices = list(
 		/obj/item/weapon/reagent_containers/food/drinks/coffee = 20,
 		/obj/item/weapon/reagent_containers/food/drinks/tea = 20,
-		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 22
+		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 22,
+		/obj/item/weapon/reagent_containers/food/snacks/donut/normal = 18
```

Comment:
Uhm.. just a suggestion. Maybe price is too high? The coffee is usually around 1.21$ in US and donuts could be as cheap as 69 cents. So donuts are 1.75 times less of a price then coffee. Here they are 0.1 times less, which is very close.

Reply:
I'll fix it. It's kinda hard to get a firm grasp on pricing without persistent economy

Reply:
That is very true